sources/service-kubernetes/main.tf

```diff
@@ -32,10 +32,20 @@ locals {
   docker_image_active  = var.docker_image_active == "" ? local.docker_image_active_calculated : var.docker_image_active
   docker_image_passive = var.docker_image_passive == "" ? local.docker_image_passive_calculated : var.docker_image_passive
 
-  ingresses_passive = [for x in var.ingresses : {
+  ingresses_passive_tmp = [for x in var.ingresses : {
     annotations = x.annotations
     rules = [for y in x.rules : {
-      host                        = "passive.${y.host}",
+      host_tmp                    = split(".", y.host),
+      host                        = y.host,
+      path                        = y.path,
+      tls_certificate_secret_name = y.tls_certificate_secret_name
+    }]
+  }]
+
+  ingresses_passive = [for x in local.ingresses_passive_tmp : {
+    annotations = x.annotations
+    rules = [for y in x.rules : {
+      host                        = join(".", concat(["${substr(y.host_tmp[0], 0, 56)}-passive"], length(y.host_tmp) > 1 ? slice(y.host_tmp, 1, length(y.host_tmp)) : []))
```

Comment:
Can we add a comment above this line that clarifies why `56` is used as the substring length?